apps/web/package.json

```diff
@@ -9,7 +9,6 @@
     "clean": "rm -rf .turbo && rm -rf node_modules && rm -rf .next && rm -rf .swc",
     "dev": "next dev --turbopack",
     "generate-swaps": "tsx test/swap/scripts/generate-swaps.ts",
-    "postinstall": "pnpm --filter \"@sushiswap/stellar-contract-binding-*\" build",
```

Comment:
Do things still work without the `postinstall` script or do we need to update a config for `turbo`?